api/webhook.go

```diff
@@ -618,6 +639,8 @@ func PostWebhook(c *gin.Context) {
 
 		h.SetStatus(constants.StatusFailure)
 		h.SetError(retErr.Error())
+
+		return
```

Comment:
we probably don't want to return a build if there was an error. this also creates a conflict because this sets a 500 return code that later gets changed to a 200.